src/test/scala/io/qbeast/spark/utils/QbeastOptimizeIntegrationTest.scala

```diff
@@ -38,19 +38,19 @@ class QbeastOptimizeIntegrationTest extends QbeastIntegrationTestSpec {
       "columnsToIndex" -> "col_1,col_2",
       "cubeSize" -> "100",
       "columnStats" ->
-        """{"col_1_min": 0.0, "col_1_max": 5000.0, "col_2_min": 0.0, "col_2_max": 5000.0}""")
+        """{"col_1_min": 0.0, "col_1_max": 10000.0, "col_2_min": 0.0, "col_2_max": 10000.0}""")
     spark
-      .range(5000)
-      .withColumn("col_1", rand() % 5000)
-      .withColumn("col_2", rand() % 5000)
+      .range(10000)
```

Comment:
I don't get this change....

Reply:
The number of records was too small for a sampling error tolerance of 5%.

Reply:
mmm, this size of the datasets thing in tests is getting weird. Do you know why this happens now and not before?

Reply:
Mm, I'm not sure if we ran this test 50 times before... But seems like it's being experienced more regularly, and I was wondering if that is because of something we changed in the middle. 

Perhaps we should rethink the use of `rand()` in the tests instead of adding more data, which would increase the test time. Anyways, we can leave it like that for this PR. I'll merge it.